src/components/Quotes.jsx

```diff
@@ -9,12 +9,13 @@ const Quotes = () => {
 
   useEffect(() => {
     const category = 'happiness';
+    const apiKey = 'Oy3qfj1OKFaF8ZyWqgnb8g==EPLPJ99iT2G8WsUL';
 
     setLoading(true);
-    axios.get('https://api.api-ninjas.com/v1/quotes', {
-      params: { category },
+
+    axios.get(`https://api.api-ninjas.com/v1/quotes?category=${category}`, {
       headers: {
-        'X-Api-Key': 'HmE/CVXQyHmYONdR03v5PA==RVOBKSCgWyZif3BP',
+        'X-Api-Key': apiKey,
```

Comment:
- [ ] Dear @shafiuyushawu, I see you have mad some changes since the last review. But as the previous reviewer mentioned your issue link is still broken. Please update a working link to your issue link inside your readme file.

Reply:
Hello @TadesseDev,

Please the link works perfectly, I believe sometimes the fetch returns an error which I suggest is from the API, not my link.

 
![image](https://user-images.githubusercontent.com/65625147/227485810-6bba56af-c13a-4c85-9ecb-c5aa460a1687.png)

Reply:
- Updated  issues link
- Fixed API request